dist/ovh-angular-otrs.js

```diff
@@ -590,10 +590,10 @@ angular.module("ovh-angular-otrs").constant("TICKET_CATEGORIES", {
             "publiccloud"
         ]
     },
-    DEFAULT: "billing"
+    DEFAULT: "billing",
+    BASEURL: "https://www.ovh.com/manager/dedicated/index.html#/ticket"
```

Comment:
works only for EU ? Right ?

We should let the possibility to override this variable